### DISCUSSION TITLE
How to localize slash command?

### DISCUSSION THREAD

**Discussion by s1upy** [discord-net/Discord.Net#2864] [Q&A] [answered]
https://discordnet.dev/guides/int_framework/intro.html#localization

How to use it in this code?
```
[SlashCommand("hello", "Say hello")]
        [EnabledInDm(false)]
        public async Task GivePremium([Summary(name: "ID", description: "Hello")] string guildId)
        { 
... 
}
```

I want this command to show up when I write /hola in Spanish.

**Comment by Misha-133** [comment, marked as answer]
Hmm, that's a great question
we didn't have any localization example before so I added resx locatization to our interaction framework sample

here's the PR, you can look at code I added
https://github.com/discord-net/Discord.Net/pull/2865

**Reply by s1upy** [reply]
How does this work in the slash command itself if you haven’t made any changes there?

**Reply by Misha-133** [reply]
It doesn't require changing anything about the command 
The values you specify on the command will be used as defaults for languages you don't make localization for, while for ones you specify it will use values from localizations
